### ISSUE TITLE
`load_cps()` fails is Arrow installation is minimal

### ISSUE THREAD

**Issue by alexpghayes** [Economic/epiextractr#14]
In this case the solution is for the user to run `arrow::install_arrow(binary = FALSE, minimal = FALSE)`, which will install a version of `arrow` that can read the `LV4` compressed version of the feather format.

**Comment by benzipperer** [comment]
yes, thank you for this solution. at a minimum we need to update the documentation to include this. maybe also a message during or after the epiextractr package installation?

(and by the way, sorry for the incomplete state of the documentation for this package)
